spec/compiler/codegen/target_spec.cr

```diff
@@ -17,6 +17,7 @@ describe Crystal::Codegen::Target do
     Target.new("i686-unknown-linux-gnu").to_s.should eq("i386-unknown-linux-gnu")
     Target.new("amd64-unknown-openbsd").to_s.should eq("x86_64-unknown-openbsd")
     Target.new("arm64-apple-darwin20.2.0").to_s.should eq("aarch64-apple-darwin20.2.0")
+    Target.new("x86_68-suse-linux").to_s.should eq("x86_68-suse-linux-gnu")
```

Comment:
Unless it was intentional, I think you meant:

```suggestion
    Target.new("x86_64-suse-linux").to_s.should eq("x86_64-suse-linux-gnu")
```

Reply:
Oops, thanks!